assets/js/blocks/products/all-products/deprecated.js

```diff
@@ -33,4 +33,29 @@ const v1 = {
 	},
 };
 
-export default [ v1 ];
+const v2 = {
+	save( { attributes } ) {
```

Comment:
Pretty sure we need to add the attributes object here just like in v1

Reply:
Updated v2 with attributes object just like in v1